src/System.ComponentModel.EventBasedAsync/tests/BackgroundWorkerTests.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+public class BackgroundWorkerTests
+{
+    private static ManualResetEventSlim manualResetEvent1 = new ManualResetEventSlim(false);
+    private static ManualResetEventSlim manualResetEvent2 = new ManualResetEventSlim(false);
+    private static ManualResetEventSlim manualResetEvent3 = new ManualResetEventSlim(false);
+
+    private static void Wait(int milliseconds)
+    {
+        System.Threading.Tasks.Task.Delay(milliseconds).Wait();
```

Comment:
System.Threading.Tasks is being imported above... shouldn't need the full namespace here.